.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.idea/
```

Comment:
That .idea/ directory is generated by PyCharm/WebStorm and IMHO is a good thing to have in .gitignore.